docs/dev/test-configuration-options.md

```diff
@@ -727,6 +727,11 @@ See [Using Tunnel Names](/secure-connections/sauce-connect/setup-configuration/b
 "tunnelIdentifier": "MyTunnel01"
 ```
 
+:::warning
+BREAKING CHANGE
+Appium tests for the Real Device Cloud using the W3C protocol MUST use tunnelName instead of tunnelIdentifier.
+:::
```

Comment:
```suggestion
:::caution Breaking Change
Appium tests for the Real Device Cloud using the W3C protocol MUST use `tunnelName` instead of `tunnelIdentifier`.
:::
```